cartesian/pfss_cartesian_obj.py

```diff
@@ -0,0 +1,133 @@
+import warnings
+
+import numpy as np
+import sunpy.map
+from astropy.io import fits
+import utils
+
+class Input:
+    r"""
+    Input to PFSS modelling.
+
+    This code uses a Cartesian geometry (x,y,z) for performing the extrapolation. 
+    Thus, make sure the grid defined is cartesian, and that solutions obtained are 
+    valid at all. 
+    Parameters
+    ----------
+    bz : sunpy.map.GenericMap
+        Boundary condition of LOS magnetic field at the inner surface.
+
+    Z_max : float
+        Max height in vertical direction to which extrapolation is performed, in Mm (Mega meters)
+
+    dz : float
+        Grid spacing. 
```

Comment:
This should be the "Grid spacing as a fraction of the {x, y} grid spacing", and I think it makes sense to make it optional and default to 1?

Another question I had, was if the cell spacing in the x and y directions has to be the same for the solver, or if it can be different?

Reply:
The grid spacing can be different in x, y and z. Since the implementation is based on number of  Fourier modes, the grid spacing can be accounted for by changing spacing in frequency space.